### PR TITLE
Allow same file to be packaged more than once

### PR DESCRIPTION
**Pull request by muuki88**
Allow files to be packaged more than once. This works now as expected

```scala
mappings ++= Seq(
  foo -> "foo1",
  foo -> "foo2"
)
```

### CODE REVIEW

File: main-actions/src/main/scala/sbt/Package.scala
Comment (muuki88) [inline main-actions/src/main/scala/sbt/Package.scala:84]: This was the actual fix

**Review by eed3si9n — APPROVED** [review]
LGTM pending CI

**Comment by muuki88** [comment]
Should I merge this? What's the process for integrating this into the 1.1.x and 1.2.x branches?

**Comment by eed3si9n** [comment]
There's actually now a debate about whether we should merge to 1.x or 1.2.x first. Let's hold off on this for a day.

**Comment by muuki88** [comment]
@eed3si9n I'll reopen this against `1.2.x` then, right?

**Comment by eed3si9n** [comment]
@muuki88 I think so.

File: main-actions/src/main/scala/sbt/Package.scala
Comment (eed3si9n) [reply main-actions/src/main/scala/sbt/Package.scala:84]: An interesting effect the use of `Map` had was deduplicating the input mapping.
https://github.com/sbt/sbt/issues/4889

**Pull request merged**
